src/scenes/Event/scenes/oracle.js

```diff
@@ -357,7 +356,7 @@ class OraclePage extends React.Component {
             message: intl.formatMessage(messages.eventUnconfirmed),
           },
         };
-      } else if (token === Token.Qtum && status === OracleStatus.Voting) {
```

Comment:
revert this change. we need `Token` to differentiate between a Centralized Oracle vs Decentralized Oracle. 

`status === voting` can be both for COracle and DOracle. (`Voting` is a synonym for `Betting`. Frank just decided to reuse the status.)

Reply:
it looks like this entire block is wrapped in `if (oracle && oracle.token === Token.Qtum)` which is incorrect. 

`else if (token === Token.Bot && status === OracleStatus.Voting)` and `else if (token === Token.Bot && status === OracleStatus.WaitResult)` will never be reached (the last two cases).

Reply:
sorry about that. My bad 😵